packages/dds/register-collection/src/consensusRegisterCollection.ts

```diff
@@ -343,6 +395,16 @@ export class ConsensusRegisterCollection<T>
 		return serializer.parse(content);
 	}
 
+	/**
+	 * @sealed
```

Comment:
Semantic docs?

Reply:
Added an inheritDoc link in latest (not sure if this is exactly what you meant?)